README.md

```diff
@@ -1,4 +1,4 @@
-# Ekalaamu
+# Ekalaamu [![Build Status](https://travis-ci.org/katunold/ekalaamu.svg?branch=develop)](https://travis-ci.org/katunold/ekalaamu) [![Maintainability](https://api.codeclimate.com/v1/badges/809dc68f4613ec441db8/maintainability)](https://codeclimate.com/github/katunold/ekalaamu/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/809dc68f4613ec441db8/test_coverage)](https://codeclimate.com/github/katunold/ekalaamu/test_coverage)
```

Comment:
some of theses bugdes you have added are not showing, is there a problem

Reply:
Yes @kdaglas and this is because of the default branch at which codeclimate and travis are pointing at is develop so all badges will display when the PR is merged to develop